MAUI/ImageEditor/save.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Save in .NET MAUI ImageEditor control | Syncfusion
```

Comment:
Image save in .NET MAUI ImageEditor control | Syncfusion
description: Learn here all about save support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.

Reply:
Modified a content